src/tv2-common/actions/executeAction.ts

```diff
@@ -1373,20 +1400,19 @@ function findPrimaryPieceUsingPriority<
 	return undefined
 }
 
-function applyPrerollToWallGraphics<
+async function applyPrerollToWallGraphics<
 	StudioConfig extends TV2StudioConfigBase,
 	ShowStyleConfig extends TV2BlueprintConfigBase<StudioConfig>
 >(
 	context: ITV2ActionExecutionContext,
 	settings: ActionExecutionSettings<StudioConfig, ShowStyleConfig>,
-	piecesBySourceLayer: PiecesBySourceLayer,
-	partProps: Partial<IBlueprintPart>
+	piecesBySourceLayer: PiecesBySourceLayer
 ) {
```

Comment:
I guess it doesn't? Do we want to remove it?

Reply:
Assuming that the prerolls set on the server pieces are working as expected, we should no longer have to delay walls manually, so It seems to me like this can go